docs/platform/release-notes/tizen-6-0-m2.md

```diff
@@ -1,4 +1,4 @@
-﻿# Tizen 6.0 Public M2 Release Notes
+﻿# Tizen 6.0 Public M2
```

Comment:
Exactly, there seems to be **\ufeff** character in front of the # Tizen 6.0 Public M2